smssync/src/main/java/org/addhen/smssync/presentation/service/SmsReceiverService.java

```diff
@@ -58,9 +58,6 @@
 public class SmsReceiverService extends Service implements HasComponent<AppServiceComponent> {
 
     @Inject
-    PostMessage mProcessMessage;
-
-    @Inject
     FileManager mFileManager;
```

Comment:
This should be annotated with `@Inject` to get the `FileManager` object by the DI